script/sync_prod_to_gcp/submissions_to_gcp.py

```diff
@@ -773,26 +773,31 @@ def submission_message_to_file_state(data: dict, log_extra: dict, ask_webnode: b
                     WEBNODE_REQUEST_COUNT = (WEBNODE_REQUEST_COUNT + 1) % n_webnodes
                     host, n_para = CONCURRENCY_PER_WEBNODE[min(n_webnodes - 1, max(0, my_tag))]
                     protocol = "http" if host.startswith("localhost:") else "https"
+                    log_extra['webnode'] = host
```

Comment:
I see that it is difficult to get the URL used to request the PDF due to  ensure_pdf signature. :+1: